realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -192,7 +192,7 @@ public static synchronized void init(Context context) {
             defaultConfiguration = new RealmConfiguration.Builder(context).build();
             ObjectServerFacade.getSyncFacadeIfPossible().init(context);
             BaseRealm.applicationContext = context.getApplicationContext();
-            SharedRealm.initialize(new File(context.getFilesDir(), ".realmNamedPipes"));
+            SharedRealm.initialize(new File(context.getFilesDir(), ".realm.temp"));
```

Comment:
@cmelchior named pipe files created in it has the hash of corresponding realm file's path in its name. Hash can collide, but I think it is out of scope of this PR.

Back up and restore is also file since the named pipe does not contain anything in it.